### PR TITLE
LP-2348/LP-2349 updated react js for password confirmation form

### PR DESCRIPTION
**Pull request by asadali145**

### CODE REVIEW

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/490/workflows/workflows_id/jobs/490/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
No lines with coverage information in this diff.
-------------
```

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/491/workflows/workflows_id/jobs/491/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
No lines with coverage information in this diff.
-------------
```

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/492/workflows/workflows_id/jobs/492/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
No lines with coverage information in this diff.
-------------
```

File: webpack.common.config.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ module.exports = Merge.smart({
                                                'program_enrollments/index.jsx',
             ProgramEnrollmentsInspectorPage: './lms/djangoapps/support/static/support/jsx/' +
                                                'program_enrollments/inspector.jsx',
-            PasswordResetConfirmation: './lms/static/js/student_account/components/PasswordResetConfirmation.jsx',
+            PasswordResetConfirmation: '/edx/app/edxapp/edx-platform/themes/adg-edx-theme/edx-platform/adg/lms/static/js/student_account/components/PasswordResetConfirmation.jsx',
```
Comment (ahmed-faraz46) [inline webpack.common.config.js:91]: Let's please use COMPREHENSIVE_THEME_DIRS settings. Because path would be different on all of our servers.

Comment (ahmed-faraz46) [inline webpack.common.config.js:91]: Let's please use COMPREHENSIVE_THEME_DIRS settings. Because path would be different on all of our servers.

**Review by ahmed-faraz46 — CHANGES_REQUESTED** [review]

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/494/workflows/workflows_id/jobs/494/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
No lines with coverage information in this diff.
-------------
```

**Review by aliassad — APPROVED** [review]
LGTM 👍

**Review by ahmed-faraz46 — APPROVED** [review]

**Pull request merged**